compiler/rustc_codegen_gcc/src/builder.rs

```diff
@@ -898,26 +898,14 @@ impl<'a, 'gcc, 'tcx> BuilderMethods<'a, 'tcx> for Builder<'a, 'gcc, 'tcx> {
         self.gcc_checked_binop(oop, typ, lhs, rhs)
     }
 
-    fn alloca(&mut self, ty: Type<'gcc>, align: Align) -> RValue<'gcc> {
-        // FIXME(antoyo): this check that we don't call get_aligned() a second time on a type.
-        // Ideally, we shouldn't need to do this check.
-        let aligned_type = if ty == self.cx.u128_type || ty == self.cx.i128_type {
-            ty
-        } else {
-            ty.get_aligned(align.bytes())
-        };
```

Comment:
Did you check that this code is indeed not needed anymore, i.e. that the relevant tests pass without this code?

Reply:
Now that the `ty` argument is gone, this code is trivially dead--`let ty = self.cx.type_array(...)` can never be u128 or i128